pymc/distributions/discrete.py

```diff
@@ -972,11 +972,11 @@ class HyperGeometric(Discrete):
     Parameters
     ----------
     N : integer
```

Comment:
are scalars the only valid type for all 3 arguments? cc @ricardoV94 ? (no idea who knows, feel free to tag someone else)

Reply:
They should be only integers: https://en.wikipedia.org/wiki/Hypergeometric_distribution

Reply:
but maybe an array of integers is also supported? :thinking:

Reply:
logcdf does say:
```
        # logcdf can only handle scalar values at the moment
        if np.ndim(value):
            raise TypeError(
                f"HyperGeometric.logcdf expects a scalar value but received a {np.ndim(value)}-dimensional object."
            )
```

Reply:
But maybe logp supports it?

Reply:
Arrays/ tensors should always assumed to be supported unless otherwise specified (as in the logcdf) or some multivariate distributions

Reply:
Should I specify the types inside the array? Example `integer, array_like[integer] or TensorVariable[integer]`?

Reply:
no, numpydoc formatting is very clear in the `array of type` format

Reply:
We decided to use `tensor_like of integer` and we'll add its definition on the glossary so we have a type that covers scalars, arrays, aesara tensorvariable and randomvariable

Reply:
Notes! Picked `integer, array_like or TensorVariable`

Reply:
Sorry for the late review. We decided to go with `tensor_like of integer` which will get rendered as a link to our glossary where we have an explanation on what inputs are accepted for arguments documented like this: https://docs.pymc.io/en/latest/glossary.html#term-tensor_like